Sources/NIOCore/CircularBuffer.swift

```diff
@@ -222,6 +222,33 @@ extension CircularBuffer: Collection, MutableCollection {
             return end.backingIndex &- start.backingIndex
         }
     }
+
+    @inlinable
+    public __consuming func _copyContents(
```

Comment:
Let's drop the `__consuming` annotation here, as it's non-stable.

Reply:
Done.